src/components/vanilla/charts/BubbleMapChart/BubbleMapChart.emb.ts

```diff
@@ -0,0 +1,133 @@
+import { OrderBy, isDimension, isMeasure, loadData } from '@embeddable.com/core';
+import { EmbeddedComponentMeta, Inputs, defineComponent } from '@embeddable.com/react';
+
+import Component from './index';
+
+export const meta = {
+  name: 'BubbleMapChart',
+  label: 'Bubble Map Chart',
+  classNames: ['inside-card'],
+  category: 'Charts: essentials',
+  inputs: [
+    {
+      name: 'ds',
+      type: 'dataset',
+      label: 'Map Dataset',
+      description: 'Dataset',
+      defaultValue: false,
+      category: 'Chart Data',
+    },
+    {
+      name: 'bubblePlacementLat',
+      type: 'dimension',
+      label: 'Bubble Placement - Latitude',
+      config: {
+        dataset: 'ds',
```

Comment:
Do we have a latitude type?

I"m guessing not, but if so, we could do something like:

supportedTypes: ['latitude']

Reply:
Unfortunately, Cube doesn't provide a latitude or longitude type